Jenkinsfile

```diff
@@ -418,7 +416,7 @@ pipeline {
                                 build job: 'zeebe-docker', parameters: [
                                     string(name: 'BRANCH', value: env.BRANCH_NAME),
                                     string(name: 'VERSION', value: env.VERSION),
-                                    booleanParam(name: 'IS_LATEST', value: isMasterBranch),
+                                    booleanParam(name: 'IS_LATEST', value: false),
```

Comment:
:question: I'm assuming we will change that later to `main`?

Reply:
Probably not, because this would cause any push to main (which was up to now develop) to be tagged as the `latest` image, which we don't want. I'm actually not sure why this was previously the case :confused: